src/components/activities/ActivityCard.jsx

```diff
@@ -193,22 +199,39 @@ class ActivityCard extends Component {
         />
       );
     }
-    return (
-      <div className='verifyButtons'>
-        <Button
-          name='approve'
-          value='Approve'
-          className='verifyButtons__button verifyButtons__button--approve'
-          onClick={() => handleClick(APPROVE, id)}
-        />
+    if (showCompleteButton) {
+      showCompleteButtonHtml = (
         <Button
-          name='reject'
-          value='Reject'
-          className='verifyButtons__button verifyButtons__button--reject'
-          onClick={() => handleClick(REJECT, id)}
+          name='complete'
+          value='Complete'
+          className='verifyButtons__button verifyButtons__button--moreInfo'
+          onClick={() => handleClick(COMPLETE, id)}
         />
-        { moreInfoButtonHtml }
-      </div>
+      );
+    }
+    return (
+      <Fragment>
+        {
```

Comment:
Why not have this as the same logic for `{moreInfoButtonHtml}` where you evaluate the result outside and then call the variable here?